Formula/polynote.rb

```diff
@@ -0,0 +1,25 @@
+class Polynote < Formula
+  desc "The polyglot notebook with first-class Scala support"
+  homepage "https://polynote.org/"
+  url "https://github.com/polynote/polynote/releases/download/0.2.8/polynote-dist.tar.gz"
+  version "0.2.8"
+  sha256 "97d15fb866269885d79218e7ed97fbe188b88d0afbf4933d7d2a6b452a857c29"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install Dir["*"]
+
+    (bin/"polynote").write <<~EOS
+      #!/usr/bin/env bash
+      (cd #{libexec} ; ./polynote)
+    EOS
+  end
+
+  test do
+    assert_predicate bin/"polynote", :exist?
+    assert_predicate bin/"polynote", :executable?
```

Comment:
This is still not a sufficient test. Surely there is a way of testing some functionality.